### ISSUE TITLE
push 6.15 to the aur

### ISSUE THREAD

**Issue by nicman23** [M-Reimer/wine-lol#87]
is there a reason it is being kept back?

**Comment by M-Reimer** [comment]
I don't use 6.15. 5.18 is up to date here.

**Comment by nicman23** [comment]
oh so the 6.15 is not up to date?

**Comment by M-Reimer** [comment]
Since the very beginning of this whole "patchy" situation there were two solutions.

The first one, which is maintained here, which requires a patch to glibc (maintained here as "wine-lol-glibc")

And the second one, which requires you to disable "vsyscall32" globally.

As I don't want to change a global setting (and especially one which is meant to improve 32 bit application performance) just to play one game, I decided to make the glibc variant more usable.

And the current situation is now that there exists an updated patch for 6.15 which allows you to use it if you disable vsyscall32 globally and all I was able to get to work is version 5.18 which, as always, uses the "glibc approach" together with wine-lol-glibc.

But as mentioned in my README:
https://github.com/M-Reimer/wine-lol#contributor-notes-about-wine-version-updates
I won't accept requests of Wine updates as Issue. If you manage to get the glibc-based patch to work with 6.15 and provide this as **tested** pull request, then I would be more than happy to merge it.
